src/sentry/web/frontend/debug/mail.py

```diff
@@ -533,6 +534,39 @@ def digest(request):
         state["event_counts"][perf_group.id] = random.randint(10, 1e4)
         state["user_counts"][perf_group.id] = random.randint(10, 1e4)
 
+    # add in generic issues
```

Comment:
At some point this whole function should be refactored, but since it affects the extremely flaky [email acceptance tests](https://github.com/getsentry/sentry/blob/master/tests/acceptance/test_emails.py) I'm reluctant to do so. Maybe a random.choice where the choices are error, performance, and generic issue would clean things up a lot.